test/WormholeL2VoteAggregator.t.sol

```diff
@@ -169,8 +170,9 @@ contract CastVote is L2VoteAggregatorTest {
     uint256 proposalId,
     uint64 timeToProposalEnd
   ) public {
-    vm.assume(_support < 2);
     vm.assume(_amount != 0);
+    vm.assume(_support < 2);
```

Comment:
i didn't understand what this `assume` block was doing. now i know we're limiting the enum index. for readability, i might define `L2VoteAggregator.VoteType _voteType = L2VoteAggregator.VoteType(_support)` right below this assume block, and then use `_voteType` down in the `castVote` call

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/53/commits/5663e3671ad135cf0453fd0e28637ce5fed6e892
https://github.com/ScopeLift/l2-flexible-voting/pull/53/commits/a103b1dce7eea3bc00028df4ba5b3a8a312dac47